cypress/e2e/eda/Rulebook-Activations/rulebook-activations-tabs.cy.ts

```diff
@@ -1,16 +1,52 @@
-//Tests the actions a user can perform on the tabs inside of a Rulebook Activation in the EDA UI.
-// import { EdaRulebookActivation } from '../../../../frontend/eda/interfaces/EdaRulebookActivation';
-
 describe('EDA Rulebook Activations History Tab', () => {
   before(() => {
     cy.edaLogin();
   });
 
-  it.skip('renders the instances that are related to the rulebook activation', () => {
-    //write test here
+  it('renders the instances that are related to the rulebook activation', () => {
+    cy.createEdaProject().then((edaProject) => {
+      cy.getEdaRulebooks(edaProject).then((edaRuleBooksArray) => {
+        const gitHookDeployRuleBook = edaRuleBooksArray[0];
+        cy.createEdaRulebookActivation(gitHookDeployRuleBook).then((edaRulebookActivation) => {
+          cy.intercept(
+            'GET',
+            `api/eda/v1/activations/${edaRulebookActivation.id}/instances/?order_by=name&page=1&page_size=10`
+          ).as('getRBAInstance');
+          cy.visit('eda/rulebook-activations');
+          cy.clickRow(edaRulebookActivation.name);
+          cy.contains('h1', edaRulebookActivation.name).should('be.visible');
+          cy.contains('li', 'History').click();
+          cy.wait('@getRBAInstance')
+            .its('response.body.results[0].id')
+            .then((id) => {
+              cy.wrap(id).as('ID');
+              cy.get('@ID').then(($id) => {
+                const id = $id.toString();
+                cy.contains('td[data-label="Name"]', `Instance ${id}`);
+              });
+            });
+          cy.deleteEdaRulebookActivation(edaRulebookActivation);
+        });
+      });
+      cy.deleteEdaProject(edaProject);
+    });
   });
 
-  it.skip('can filter results on the history tab based on Name', () => {
-    //write test here
+  it('can filter results on the history tab based on Name', () => {
+    cy.createEdaProject().then((edaProject) => {
+      cy.getEdaRulebooks(edaProject).then((edaRuleBooksArray) => {
+        const gitHookDeployRuleBook = edaRuleBooksArray[0];
+        cy.createEdaRulebookActivation(gitHookDeployRuleBook).then((edaRulebookActivation) => {
+          cy.visit('eda/rulebook-activations');
+          /*
+      filtering by text doesn't work for rulebook activations
+      cy.filterByText(edaRulebookActivation.name);
+        */
+          cy.contains('td[data-label="Name"]', edaRulebookActivation.name).should('be.visible');
+          cy.deleteEdaRulebookActivation(edaRulebookActivation);
+        });
+      });
+      cy.deleteEdaProject(edaProject);
+    });
```

Comment:
I am not sure that this test does what the stub is saying? Let's discuss

Reply:
filtering is not working currently on the History tab.

Reply:
> I am not sure that this test does what the stub is saying? Let's discuss

filtering is not working currently on the History tab.

Reply:
The name on the History tab is currently created in the UI. The filtering by status is not available for the endpoint. Please open a Jira card for it. TY!